pkgs/build-support/src-only/default.nix

```diff
@@ -1,22 +1,37 @@
-{ stdenv }:
+{ lib, stdenvNoCC }:
+
 # srcOnly is a utility builder that only fetches and unpacks the given `src`,
 # and optionally patching with `patches` or adding build inputs.
 #
 # It can be invoked directly, or be used to wrap an existing derivation. Eg:
 #
 # > srcOnly pkgs.hello
 #
```

Comment:
While you're in the area, Gavin (@Pandapip1), do you might re-formatting this so that it shows up on Noogle?

https://github.com/NixOS/nixpkgs/pull/332466/files is a great example of how to do it.

CC @hsjobeki